radiobuttons/src/main/java/com/decathlon/vitamin/compose/radiobuttons/VitaminRadioButtons.kt

```diff
@@ -65,7 +65,6 @@ object VitaminRadioButtons {
                 )
                 .semantics(mergeDescendants = true) {}
                 .sizeIn(
-                    minWidth = sizes.minimumSize,
```

Comment:
We do need these min dimensions as it is the minimum required for a clickable element to respect accessibility. 
We can add 
```horizontalArrangement = Arrangement.Center,```

Reply:
yes you're right thank you !

Reply:
And I need to add this into the template to be check, but don't forget to re generate snapshots for paparazzi UI tests with `./gradlew radiobuttons:recordPaparazziDebug` as you changed UI. Otherwise, tests will fail (./gradlew verifyPaparazziDebug)

Reply:
done :D thx !